### PR TITLE
280 syncing lot of logs isnt reliable with poor internet connections

### PR DESCRIPTION
**Pull request by jmbrunskill**
- [ ] HTTP Timeout is now 300,000ms (10 minutes) to allow for slow syncing behaviour.
- [ ] Batch size set to 100 records
- [ ] No longer sends batches concurrently
- [ ] Sync Error is shown in Settings

image

### CODE REVIEW

File: src/features/Sync/SyncSlice.ts [hunk 1]
```diff
@@ -292,11 +289,15 @@ function* syncAll({
         SyncAction.syncTemperatureBreachesSuccess,
         SyncAction.syncTemperatureBreachesFailure,
       ]);
+      yield put(SettingAction.update('lastSync', utils.now()));
     } catch (e) {
       console.error('syncAll error', e);
+      yield put(SyncAction.updateSyncError((e as Error).message));
```
Comment (jmbrunskill) [inline src/features/Sync/SyncSlice.ts:295]: I don't think this will ever trigger, but just incase it doesn't hurt I guess.

**Review by mark-prins — APPROVED** [review]
🎉  that's much better thanks
I've committed my suggestion in c79566c rather than create a suggestion branch. see what you think - and fix as needed..

**Comment by jmbrunskill** [comment]
> 🎉 that's much better thanks I've committed my suggestion in [c79566c](https://github.com/msupply-foundation/msupply-cold-chain/commit/c79566c9a0d6cb1c7c0b8428be9647637b7de736) rather than create a suggestion branch. see what you think - and fix as needed..

Yeah I like this approach, much easier than a second branch!

File: src/common/constants/Timeout.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
 export const TIMEOUT = {
   LOGIN_TIMEOUT: 20000,
-  HTTP_TIMEOUT: 60000,
+  HTTP_TIMEOUT: 300000, // 10 minutes to sync
 };
```
[the remark this answers is not in the document]
Comment (jmbrunskill) [reply src/common/constants/Timeout.ts]: Nice, yeah I think I tired 10 minutes first then decided on 5...

**Pull request merged**